cl2_test.go

```diff
@@ -28,14 +28,14 @@ func getCL2Lines(t *testing.T) [][]byte {
 	}
 	file, err := os.Open("testdata/citylots2.json.gz")
 	if err != nil {
-		t.Error("Can't open citlots.jlines.gz: " + err.Error())
+		t.Fatal("Can't open citylots2.json.gz: " + err.Error())
```

Comment:
nit: in future I suggest using `t.Fatalf("Can't open citylots2.json.gz: %v", err)` style

Reply:
agreed